plugins/auth-backend/src/providers/auth0/provider.ts

```diff
@@ -104,14 +126,17 @@ export class Auth0AuthProvider implements OAuthHandlers {
       prompt: 'consent',
       scope: req.scope,
       state: encodeState(req.state),
+      audience: this.audience as string,
```

Comment:
Hm is this to deal with undefined? Could do `...(this.audience ? { audience: this.audience } : {})` and same below. This ensures that the `audience` key isn't present at all in the object when there's no audience

Reply:
done hadn't thought of that thanks!